### PR TITLE
Quickpin ports

### PR DESCRIPTION
**Pull request by kdb424**
Porting more boards to Quickpin, and added kb2040 to the list of quickpin targets.

### CODE REVIEW

File: boards/boardsource/3x4/README.md [hunk 1]
```diff
@@ -11,3 +11,11 @@ Retailers (USA)
 
 Extensions enabled by default  
 - [Layers](/docs/layers.md) Need more keys than switches? Use layers.
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/boardsource/3x4/README.md:17]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/boardsource/4x12/README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 ![Boardsource4x12](https://boardsource.imgix.net/164c3388-5057-46c8-8fcd-82c58c7870ce.jpg?raw=true)
 ![Boardsource4x12LP](https://boardsource.imgix.net/c2108ea4-7d70-4327-b4b4-88c8191b1369.jpg?raw=true)
 
-The 4x12 ortholinear keyboard is an extremely common and beloved layout within the keyboard community. Made popular by Jack's Planck from OLKB, the 4x12 ortholinear layout is possibly the most popular non-stagger layout available.  
-
-kb.py is designed to work with the nice!nano  
+The 4x12 ortholinear keyboard is an extremely common and beloved layout within
+the keyboard community. Made popular by Jack's Planck from OLKB, the 4x12
+ortholinear layout is possibly the most popular non-stagger layout available.
```
Comment (xs5871) [inline boards/boardsource/4x12/README.md:8]:
```suggestion
ortholinear layout is possibly the most popular non-staggered layout available.
```

File: boards/boardsource/4x12/README.md [hunk 3]
```diff
@@ -19,3 +19,10 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/boardsource/4x12/README.md:25]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/boardsource/5x12/README.md [hunk 4]
```diff
@@ -19,3 +22,10 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/boardsource/5x12/README.md:28]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/boardsource/microdox/README.md [hunk 5]
```diff
@@ -17,3 +17,11 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/boardsource/microdox/README.md:23]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/crkbd/README.md [hunk 6]
```diff
@@ -25,3 +23,11 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/crkbd/README.md:29]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/ergo_travel/README.md [hunk 7]
```diff
@@ -22,3 +28,12 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/ergo_travel/README.md:35]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/fingerpunch/ffkb/README.md [hunk 8]
```diff
@@ -2,12 +2,24 @@
...
+> steps. See guidance [over
+> here](../../docs/Officially_Supported_Microcontrollers.md#nicenano).
+
+Use `kb2040/kb.py` when using any other pro micro footprint MCU.
 
 An example `main.py` file is included for each MCU.
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/fingerpunch/ffkb/README.md:21]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/fourtypercentclub/gherkin/README.md [hunk 9]
```diff
@@ -15,3 +13,11 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/fourtypercentclub/gherkin/README.md:19]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/fourtypercentclub/luddite/README.md [hunk 10]
```diff
@@ -16,3 +13,11 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/fourtypercentclub/luddite/README.md:19]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/keebio/iris/README.md [hunk 11]
```diff
@@ -13,3 +12,10 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/keebio/iris/README.md:18]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/keebio/levinson/README.md [hunk 12]
```diff
@@ -13,3 +12,11 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/keebio/levinson/README.md:18]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/keebio/nyquist/README.md [hunk 13]
```diff
@@ -14,3 +13,10 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/keebio/nyquist/README.md:19]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/kyria/README.md [hunk 14]
```diff
@@ -51,3 +49,11 @@ Because of the above, besides of normal installation steps, you have to also:
 
 - install Circuit Python in 7.2+ version
 - add `adafruit_pioasm.mpy` library to lib or root folder of a board
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/kyria/README.md:55]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/lily58/README.md [hunk 15]
```diff
@@ -18,3 +20,11 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/lily58/README.md:26]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/reviung39/README.md [hunk 16]
```diff
@@ -18,3 +21,11 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/reviung39/README.md:27]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/reviung41/README.md [hunk 17]
```diff
@@ -18,3 +22,10 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/reviung41/README.md:28]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/rhymestone/README.md [hunk 18]
```diff
@@ -17,3 +19,11 @@ Extensions enabled by default
 Common Extensions
 - [Split](/docs/split_keyboards.md) Connects halves using a wire
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/rhymestone/README.md:25]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

File: boards/tg4x/README.md [hunk 19]
```diff
@@ -15,3 +15,11 @@ Extensions enabled by default
 
 Common Extensions
 - [Power](/docs/power.md) Powersaving features for battery life
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`
```
Comment (xs5871) [inline boards/tg4x/README.md:21]:
```suggestion
Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
```

**Review by xs5871 — APPROVED** [review]

**Comment by kdb424** [comment]
Applied all suggestions, squashed, and pushed.

**Pull request merged**